models/biocro/R/run.biocro.R

```diff
@@ -138,6 +139,8 @@ run.biocro <- function(lat, lon, metpath, soil.nc = NULL, config = config, coppi
   daily.results <- hourly.results[, list(Stem = max(Stem), 
                                          Leaf = max(Leaf),
                                          Root = max(Root), 
+                                         AboveLitter = max(AboveLitter),
```

Comment:
I wondered that too -- presumably to guarantee each var has length 1? BioCro will happily produce multiple lines per hour if given sub-hourly input, but maybe that's a pathology that should be caught further upstream.